mul_benchmark.py

```diff
@@ -0,0 +1,41 @@
+#! /bin/bash/python3
+# -*- coding: utf-8 -*-
+import torch
+import torch.utils.benchmark as benchmark
+
+def generate_example(device, dtype):
+    a = torch.ones(100, device=device, dtype=dtype)
+    b = torch.ones(100, device=device, dtype=dtype)
+    yield (a, b)
```

Comment:
looks good! nit: since we're measuring a change to framework overhead, which we expect to be very small, you want your benchmark to be measuring mostly framework overhead and not other things. If you benchmark with larger tensors, then you might end up seeing most of the benchmark get attributed to the time spent in the `mul` kernel, and not the actual framework overhead.

You can see the difference more easily if you use really tiny tensors